src/how-to-contribute.md

```diff
@@ -31,6 +31,12 @@ You can also edit any page by clicking the "Suggest an edit" button (<i class="f
 This will start the process described above by forking the book repository.
 ```
 
+```admonish tip
+When editing Markdown content, please **write each sentence on a separate line**.
```

Comment:
Should we consider rewording this to 
When editing Markdown content, please start each sentence on a new line?
Feel like this is less ambiguous

Reply:
Yep, that makes it clear. Good point!

Reply:
I'm trying to figure out if the UI can create the commit for us, or if I need to do it myself ...